zwave_js_server/model/endpoint.py

```diff
@@ -0,0 +1,44 @@
+"""
+Model for a Zwave Node's endpoints.
+
+https://zwave-js.github.io/node-zwave-js/#/api/endpoint?id=endpoint-properties
+"""
+
+from typing import Optional, TypedDict
+
+
+class EndpointDataType(TypedDict, total=False):
+    """Represent an endpoint data dict type."""
+
+    nodeId: int
+    index: int
+    installerIcon: Optional[int]
```

Comment:
Mark the others required with a comment instead. I'm assuming the optional items will not be present at all in the data dict if unset. `Optional` means they can be `None` which may not be the case.

Reply:
yes that's correct